std_msgs/Header    header

bool               target_visible

float32            top_left_x
float32            top_left_y
float32            width
float32            height

int32              image_width
int32              image_height

string             class_name
float32            class_prob